### PR TITLE
[zh-cn]: Update translation of String.lastIndexOf()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13975

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/string/lastindexof/index.md [hunk 1]
```diff
@@ -5,72 +5,71 @@ slug: Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
...
+
+- `position` {{optional_inline}}
+
+  - : 该方法返回指定子字符串在小于或等于 `position` 的位置中的最后一次出现的索引，默认为 `+Infinity`。如果 `position` 大于调用字符串的长度，则该方法将搜索整个字符串。如果 `position` 小于 `0`，则行为与 `0` 相同，即该方法只在索引 `0` 处查找指定的子字符串。
+
+    - `'hello world hello'.lastIndexOf('world', 4)` 返回 `-1` — 因为虽然子字符串 `world` 在索引 `6` 处出现，但该位置不小于或等于 `4`。
+
+    - `'hello world hello'.lastIndexOf('hello', 99)` 返回 `12` — 因为小于或等于 `99` 的位置中，最后一次出现 `hello` 的位置是索引 `12`。
+
+    - `'hello world hello'.lastIndexOf('hello', 0)` 和 `'hello world hello'.lastIndexOf('hello', -5)` 都返回 `0` — 因为两者都导致该方法只在索引 `0` 处查找 `hello`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/lastindexof/index.md:33]:
```suggestion
    - `'hello world hello'.lastIndexOf('world', 4)` 返回 `-1`——因为虽然子字符串 `world` 在索引 `6` 处出现，但该位置不小于或等于 `4`。

    - `'hello world hello'.lastIndexOf('hello', 99)` 返回 `12`——因为小于或等于 `99` 的位置中，最后一次出现 `hello` 的位置是索引 `12`。

    - `'hello world hello'.lastIndexOf('hello', 0)` 和 `'hello world hello'.lastIndexOf('hello', -5)` 都返回 `0`——因为两者都导致该方法只在索引 `0` 处查找 `hello`。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/lastindexof/index.md [hunk 2]
```diff
@@ -5,72 +5,71 @@ slug: Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
...
+"Blue Whale, Killer Whale".lastIndexOf("blue"); // 返回 -1
 ```
 
 ## 示例
 
-### 示例：使用 `indexOf` 和 `lastIndexOf`
+### 使用 indexOf() 和 lastIndexOf()
 
-下例使用 `indexOf` 和 `lastIndexOf` 方法来定位字符串 "`Brave new world`" 中的值。
+以下示例使用 {{jsxref("String.prototype.indexOf()", "indexOf()")}} 和 `lastIndexOf()` 在字符串 "`Brave, Brave New World`" 中查找值。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/lastindexof/index.md:66]: Would you mind to also update this in en-US :)

```suggestion
以下示例使用 {{jsxref("String.prototype.indexOf()", "indexOf()")}} 和 `lastIndexOf()` 在字符串 `"Brave, Brave New World"` 中查找值。
```

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/string/lastindexof/index.md:66]: > Would you mind to also update this in en-US :)
ok

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/string/lastindexof/index.md:66]: in https://github.com/mdn/content/pull/27933/files

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**